benchmarks/Bio/BCR.jmd

```diff
@@ -14,138 +14,322 @@ Catalyst network model to ODEs.
-          #Dict(:alg=>Rosenbrock23(autodiff=false)),
-          Dict(:alg=>TRBDF2(autodiff=false)),
-          Dict(:alg=>QNDF(autodiff=false)),
-          Dict(:alg=>FBDF(autodiff=false)),
-          Dict(:alg=>CVODE_BDF()),
-          Dict(:alg=>CVODE_BDF(linear_solver=:LapackDense)),
-          #Dict(:alg=>rodas()),
-          #Dict(:alg=>radau()),
-          #Dict(:alg=>Rodas4(autodiff=false)),
-          #Dict(:alg=>Rodas5(autodiff=false)),
-          Dict(:alg=>KenCarp4(autodiff=false)),
-          Dict(:alg=>KenCarp47(autodiff=false)),
-          #Dict(:alg=>RadauIIA5(autodiff=false)),
-          #Dict(:alg=>lsoda()),
-          ];
-```
-
-## Automatic Jacobian Solves
-
-Due to the computational cost of the problem, we are only going to focus on the
-methods which demonstrated computational efficiency on the smaller biochemical
-benchmark problems. This excludes the exponential integrator, stabilized explicit,
-and extrapolation classes of methods.
-
-First we test using auto-generated Jacobians (finite difference)
-```julia
-wp = WorkPrecisionSet(oprob,abstols,reltols,setups;error_estimate=:l2,
-                      saveat=tf/10000.,appxsol=test_sol,maxiters=Int(1e5),numruns=1)
-plot(wp)
-```
-
-## Sparse Jacobian
-Finally we test using the generated sparse analytic Jacobian function.
-```julia
-setups = [
-          #Dict(:alg=>Rosenbrock23(autodiff=false)),
-          Dict(:alg=>TRBDF2(autodiff=false)),
-          Dict(:alg=>QNDF(autodiff=false)),
-          Dict(:alg=>FBDF(autodiff=false)),
-          #Dict(:alg=>CVODE_BDF(linear_solver=:KLU)), # Fails!
-          #Dict(:alg=>rodas()),
-          #Dict(:alg=>radau()),
-          #Dict(:alg=>Rodas4(autodiff=false)),
-          #Dict(:alg=>Rodas5(autodiff=false)),
-          Dict(:alg=>KenCarp4(autodiff=false)),
-          Dict(:alg=>KenCarp47(autodiff=false)),
-          #Dict(:alg=>RadauIIA5(autodiff=false)),
-          #Dict(:alg=>lsoda()),
-          ];
-wp = WorkPrecisionSet(sparsejacprob,abstols,reltols,setups;error_estimate=:l2,
-                      saveat=tf/10000.,appxsol=test_sol,maxiters=Int(1e5),numruns=1)
-plot(wp)
+if false
+    setups = [
+            Dict(:alg=>TRBDF2(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true)),
+            Dict(:alg=>QNDF(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true)),
+            Dict(:alg=>FBDF(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true)),
+            Dict(:alg=>KenCarp4(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true)),
+            Dict(:alg=>Rosenbrock23(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true)),
+            Dict(:alg=>Rodas4(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true)),
+            Dict(:alg=>Rodas5P(linsolve=KrylovJL_GMRES(),autodiff=false,precs=incompletelu_fceri,concrete_jac=true))
+            ];

```

Comment:
Rodas and Rosenbrock don't make sense on this problem? What is this doing here?

Reply:
You are probably right, I'm not terribly familiar with the solvers, sop added Rodas (and Rosenbrock) to all the implicit cases. The idea was to then take the slow solvers and put them in a separate plot. However, I am happy to remove them entirely.